pkgs/development/compilers/gcc/common/configure-flags.nix

```diff
@@ -80,14 +98,6 @@ let
     ] ++ lib.optionals (targetPlatform.libc == "musl") [
       # musl at least, disable: https://git.buildroot.net/buildroot/commit/?id=873d4019f7fb00f6a80592224236b3ba7d657865
       "--disable-libmpx"
-    ] ++ lib.optionals crossMingw [
-      "--enable-sjlj-exceptions"
-      "--enable-hash-synchronization"
-      "--enable-libssp"
-      "--disable-nls"
```

Comment:
@angerman @Ericson2314 Were these flags moved to `crossStageStatic` (now called `withoutTargetLibc`) by accident?

Reply:
It was done in @angerman's commit here [`f2a33e4` (#89582)](https://github.com/NixOS/nixpkgs/pull/89582/commits/f2a33e4486b40434241b964be5c685600c84c3a7), with no comment